assets/javascripts/kitten/constants/typography-config.js

```diff
@@ -4,6 +4,32 @@ const TYPOGRAPHY = {
   fontWeight: 400,
   lineHeight: 1.5, // rem
   scaleMultiplier: 1.125,
+  fonts: {
```

Comment:
Vu que ce sont des styles en dessous, est-ce qu'il ne faudrait pas l'appeler `fontStyles` par exemple ? Pour bien faire la différence avec ce qu'on a au dessus avec des propriétés qui ne sont pas forcément des styles ?
```suggestion
  fontStyles: {
```

Reply:
Oui ça me va 👍